src/xenia/base/logging.cc

```diff
@@ -328,7 +331,7 @@ void FatalError(const char* fmt, ...) {
                 MB_OK | MB_ICONERROR | MB_APPLMODAL | MB_SETFOREGROUND);
   }
 #endif  // WIN32
-
+  logger_->~Logger();
```

Comment:
I don't think this belongs to this PR. What is this changing?

Reply:
See commit message, by manually destroying the logger when a fatalerror is triggered, we trigger a log flush so the fatal error actually shows up in the log file.

Reply:
It would probably do to move this change out into its own PR all on its own. Instead of being buried in unrelated code.

Reply:
Moved to a separate PR. This PR is not dependent on that fix anyway, was just a quality of life improvement